src/pykeen/triples/triples_factory.py

```diff
@@ -932,13 +941,15 @@ def from_labeled_triples(
     @classmethod
     def from_path(
         cls,
+        *,
```

Comment:
this seems to conflict with the fact that you removed the path from the usages of TriplesFactory.from_path in the examples. Typically, I still leave the first required argument before the `*` since common usage might not have the keyword

Reply:
I added the path keyword parameter

Reply:
Thanks!

Reply:
On second thought, this looks a bit redundant since the function is already called `from_path`.